### PR TITLE
fix: video not rendering in website 

### LINKED ISSUE #115: bug: Installation Video Not Working
# Describe 🐞 

The Installation Video in the [Getting Started Section](https://kubesimplify.github.io/ksctl/docs/getting-started/install) is broken.

# Reproduce 💻 ➡️  💻 

- Go to the above mentioned link
- Scroll Down and try to play the `Installation Video`

# Possible Fix & Expected behavior🔧 

Needs Discussion
https://stackoverflow.com/questions/14079808/html5-video-is-not-playing-mp4-error-invalid-source/

# Screenshots :framed_picture:
image

### PR DESCRIPTION
**Pull request by NayanPatidar**
# Tasks description :construction: 🔧 

fix Video is not rendering on website

- [x] Closes #115

# Solution :heavy_check_mark:

Changing the path to "../../videos/ksctl-install.mp4"  will work.

# Note to reviewers :notebook:

I tried changing the source to ../../static/videos/ksctl-install.mp4 but it failed. 

- [x] Ran Unit Test locally
- [x] Tested the website locally

### CODE REVIEW

**Comment by NayanPatidar** [comment]
Screenshot 2023-06-10 at 21-03-22 bug Installation Video Not Working · Issue #115 · kubesimplify_ksctl

Adding ksctl-install.mp4 to the website/docs/getting-started and changing the source path can work. Please check!!!

**Review by SaarthakMaini — COMMENTED** [review]
I think instead of adding ksctl-install.mp4 to the getting-started directory, we should try using an Absolute or Relative path to refer to the video at 'ksctl/website/static/videos/ksctl-install.mp4'

**Comment by NayanPatidar** [comment]
Sure

**Review by NayanPatidar — COMMENTED** [review]
So I made changes to the src of the video as you said. Can you check out the changes again?

**Comment by SaarthakMaini** [comment]
Looks good to me !

CC : @NayanPatidar @dipankardas011

**Comment by dipankardas011** [comment]
yes checking

**Comment by dipankardas011** [comment]
> Looks good to me !
> 
> CC : @NayanPatidar @dipankardas011

It is still not working!
image

you can test it locally
```bash
cd website
npm install
yarn start
```

@AvineshTripathi can you take a look why the video is not being rendered

**Comment by dipankardas011** [comment]
Also @NayanPatidar as you have not used the correct PR template can you copy it and make it standardized

**Comment by NayanPatidar** [comment]
ok

**Comment by NayanPatidar** [comment]
Hey so issue is solved and the correct path for it would be :-  ../../videos/ksctl-install.mp4

**Comment by dipankardas011** [comment]
@NayanPatidar Can you add the videos to their respective sections

**Comment by dipankardas011** [comment]
Yes it worked

**Comment by dipankardas011** [comment]
is their no other method to make it work other than relative address because I am not sure if the CI build will work correctly

**Comment by dipankardas011** [comment]
Can you remove the caution sign from the video ?

**Comment by dipankardas011** [comment]
> Hey so issue is solved and the correct path for it would be :- ../../videos/ksctl-install.mp4 @dipankardas011 @SaarthakMaini

can you explain why did it worked?

**Comment by dipankardas011** [comment]
also for each provider-related video you can add it to their respective markdown like for civo in providers/civo.md

and links for all of them in the getting started guide

**Comment by dipankardas011** [comment]
@AvineshTripathi should we add the videos to the bottom or at the top of the page?

**Comment by NayanPatidar** [comment]
> @NayanPatidar Can you add the videos to their respective sections

Sure I will do it.

**Comment by NayanPatidar** [comment]
Screenshot from 2023-06-12 19-07-30


Should I add the videos here ??? @dipankardas011 @SaarthakMaini

**Comment by dipankardas011** [comment]
> Screenshot from 2023-06-12 19-07-30
> 
> Should I add the videos here ??? @dipankardas011 @SaarthakMaini

I think video demo for all the providers can be added at the last of the docs

**Comment by NayanPatidar** [comment]
Added links for each provider related-videos to the Getting Started Guide
@dipankardas011 @SaarthakMaini

File: website/docs/getting-started/install.md [hunk 1]
```diff
@@ -124,13 +124,25 @@ make uninstall
 :::
 
 
-:::caution VIDEO not playable
+:::success GUIDE TO INSTALL
```
Comment (dipankardas011) [inline website/docs/getting-started/install.md:127]:
```suggestion
:::success INSTALLATION DEMO
```

File: website/docs/getting-started/install.md [hunk 2]
```diff
@@ -124,13 +124,25 @@ make uninstall
...
 
-# Installation video
+# Installation video
```
Comment (dipankardas011) [inline website/docs/getting-started/install.md:129]:
```suggestion
## ksctl install
```

File: website/docs/getting-started/install.md [hunk 3]
```diff
@@ -124,13 +124,25 @@ make uninstall
...
 
 <video width="320" height="240" controls>
-<source src="/videos/ksctl-install.mp4" type="video/mp4" />
+<source src="../../videos/ksctl-install.mp4" type="video/mp4" />
 Your browser does not support the video tag.
 </video>
 
+:::
+
+:::success GUIDE TO SUPPORT
```
Comment (dipankardas011) [inline website/docs/getting-started/install.md:138]:
```suggestion
:::success Demo for Supported Providers
```

File: website/docs/getting-started/install.md [hunk 4]
```diff
@@ -124,13 +124,25 @@ make uninstall
...
+
+##  SUPPORT FOR CIVO & AZURE
```
Comment (dipankardas011) [inline website/docs/getting-started/install.md:140]:
```suggestion
```

File: website/docs/providers/azure.md [hunk 5]
```diff
@@ -126,3 +126,25 @@ clusters which are managed by the cloud provider
...
+<video width="320" height="240" controls>
+<source src="../../videos/ksctl-azure-managed.mp4" type="video/mp4" />
+Your browser does not support the video tag.
+</video>
+
+:::
+
+:::success GUIDE
+
+## &nbsp High Avalibility Cluster - Azure {#azureHA}
```
Comment (dipankardas011) [inline website/docs/providers/azure.md:142]:
```suggestion
:::success Demo on Highly Available Cluster {#azureHA}
```

File: website/docs/providers/azure.md [hunk 6]
```diff
@@ -99,7 +99,7 @@ ksctl cred
 ## Current Features
 
 ### Cluster features
-#### High Avalibility cluster
+#### High Availability cluster
```
Comment (dipankardas011) [inline website/docs/providers/azure.md:102]:
```suggestion
#### Highly Available cluster
```

File: website/docs/intro.md [hunk 7]
```diff
@@ -86,9 +86,9 @@ Having core features of `get`, `create`, `delete`, `add nodes`, `switch`, under
 - Local
     - kind cluster with specific version
 - Civo
-    - have support for the managed and High Available clusters(_Custom_)
+    - have support for the managed and High Availability Cluster(_Custom_)
```
Comment (dipankardas011) [inline website/docs/intro.md:89]:
```suggestion
    - have support for the managed and Highly Available Cluster(_Custom_)
```

File: website/docs/intro.md [hunk 8]
```diff
@@ -86,9 +86,9 @@ Having core features of `get`, `create`, `delete`, `add nodes`, `switch`, under
...
 - Azure
-    - have support for the managed and High Available clusters(_Custom_)
+    - have support for the managed and High Availability Cluster(_Custom_)
```
Comment (dipankardas011) [inline website/docs/intro.md:91]:
```suggestion
    - have support for the managed and Highly Available Cluster(_Custom_)
```

File: website/docs/providers/civo.md [hunk 9]
```diff
@@ -40,7 +40,7 @@ ksctl cred
 ## Current Features
 
 ### Cluster features
-#### High Avalibility cluster
+#### High Availability cluster
```
Comment (dipankardas011) [inline website/docs/providers/civo.md:43]:
```suggestion
#### Highly Available cluster
```

File: website/docs/providers/civo.md [hunk 10]
```diff
@@ -67,3 +67,24 @@ clusters which are managed by the cloud provider
 - SSH access to each workplane _Private Access_ via local network, secured by private key
 :::
 
+:::success GUIDE
+
+## &nbsp Managed Cluster - Civo {#civoManaged}
```
Comment (dipankardas011) [inline website/docs/providers/civo.md:72]:
```suggestion
:::success Demo on Managed Cluster {#civoManaged}
```

File: website/docs/getting-started/install.md [hunk 11]
```diff
@@ -124,13 +124,25 @@ make uninstall
...
+:::success GUIDE TO SUPPORT
+
+##  SUPPORT FOR CIVO & AZURE  
+
+#### + [Azure High-Availability Cluster](../providers/azure.md#azureHA)
+#### + [Azure Managed Cluster](../providers/azure.md#azureManaged)
+#### + [Civo High-Availability Cluster](../providers/civo.md#civoHA)
+#### + [Civo Managed Cluster](../providers/civo.md#civoManaged)
+
+
```
Comment (dipankardas011) [inline website/docs/getting-started/install.md:147]: remove extra lines

**Comment by NayanPatidar** [comment]
Screenshot from 2023-06-14 09-56-09
##     Will this be Fine ?


Removing the header "### &nbsp &nbsp &nbsp Managed Cluster {#azureManaged}" Won't allow us to add a link to the video. I tried many different methods but its not working. I guess this should be fine

**Comment by dipankardas011** [comment]
> Screenshot from 2023-06-14 09-56-09
> 
> ## Will this be Fine ?
> Removing the header "### &nbsp &nbsp &nbsp Managed Cluster {#azureManaged}" Won't allow us to add a link to the video. I tried many different methods but its not working. I guess this should be fine

yes it LGTM

**Comment by NayanPatidar** [comment]
Changes have been made as suggested. Locally ran them and checked. Everything is running smooth.

**Comment by dipankardas011** [comment]
> Changes have been made as suggested. Locally ran them and checked. Everything is running smooth. @SaarthakMaini @dipankardas011

Why is their so much space
image

**Comment by NayanPatidar** [comment]
> > Changes have been made as suggested. Locally ran them and checked. Everything is running smooth. 
> 
> Why is their so much space image

Width and height of the video was not according to the aspect ratio of the video adding extra padding. To solve this I changed the width and height of all the videos according to the aspect ratio

**Comment by dipankardas011** [comment]
> > > Changes have been made as suggested. Locally ran them and checked. Everything is running smooth.
> > 
> > 
> > Why is their so much space image
> 
> Width and height of the video was not according to the aspect ratio of the video adding extra padding. To solve this I changed the width and height of all the videos according to the aspect ratio

telling about the Heading "Managed CLuster"

**Comment by NayanPatidar** [comment]
image

You mean this space ?

**Comment by NayanPatidar** [comment]
> > > > Changes have been made as suggested. Locally ran them and checked. Everything is running smooth.
> > > 
> > > 
> > > Why is their so much space image
> > 
> > 
> > Width and height of the video was not according to the aspect ratio of the video adding extra padding. To solve this I changed the width and height of all the videos according to the aspect ratio
> 
> was referring to the Heading "Managed CLuster" and similarly other providers have this weird tab for this

image

This must be fine right ?

**Comment by dipankardas011** [comment]
> image
> 
> You mean this space ? It would be really great if you describe the issue a little bit more.smile

yes

**Comment by NayanPatidar** [comment]
> > image
> > You mean this space ? It would be really great if you describe the issue a little bit more.smile
> 
> yes
 
Done ! Commit has been already made :+1:

**Review by dipankardas011 — APPROVED** [review]
LGTM

**Comment by NayanPatidar** [comment]
Should I delete this patch-1 branch or not?

**Pull request merged**